dotCMS/src/integration-test/java/com/dotmarketing/quartz/DotStatefulJobTest.java

```diff
@@ -110,7 +110,7 @@ public void Test_Launch_Stateful_Jobs_Verify_They_Dont_Overlap_In_Time()
     private Optional<JobExecutionContext> getJobExecutionContext(){
         final String jobName = getJobName(MyStatefulJob.class);
         try {
-            final Scheduler sequentialScheduler = getSequentialScheduler();
+            final Scheduler sequentialScheduler = DotSchedulerFactory.getInstance().getScheduler();
```

Comment:
Nitpick: Do we still need to name it `sequentialScheduler`?

Reply:
Huum, well, I think the scheduler acts as Sequential if the job is a StatefulJob and just a scheduler if the job is a Job.